src/qml/FeatureForm.qml

```diff
@@ -591,6 +592,8 @@ Page {
       color: !model.constraintsHardValid ?  Theme.errorColor : !model.constraintsSoftValid ? Theme.warningColor : Theme.mainColor
     }
 
+    Material.foreground: undefined
```

Comment:
What is this for exactly?

Reply:
The `ToolBox` enters a binding loop implicitly: 

```QML PageHeader (parent or ancestor of Material): Binding loop detected for property "foreground"```

Reply:
So I should set the `Material.foreground` either to `undefined` or color of our choise (while). I think undefined is the safer option.